.github/workflows/php.yml

```diff
@@ -0,0 +1,52 @@
+name: PHP Static Analysis & Tests
+
+on:
+  push:
+    branches: [ develop, 'issue/592' ]
+  pull_request:
+    branches: [ develop ]
```

Comment:
Why not for all pull requests?
```suggestion
```

Reply:
I suppose the only reason not to run it on PRs set against `master` is you generally don't want people doing PRs against `master` :P

Reply:
Not sure. :thinking: 
That's my first implementation of GitHub Actions from the last weekend: https://github.com/TS3Tools/zabbix-teamspeak-template/blob/main/.github/workflows/github-ci-pull_requests.yml

There... I'm doing pull requests against `main` (`master`). :D

Reply:
Let's test your solution first. We can still change this later.